<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTourDialog</class>
 <widget class="QDialog" name="CreateTourDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>376</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ایجاد اردوی جدید</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/resources/icons/addTour.png</normaloff>:/icons/resources/icons/addTour.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <layout class="QVBoxLayout" name="VLayoutMain">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="FLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblOrigin">
       <property name="text">
        <string>مبدا</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblDestination">
       <property name="text">
        <string>مقصد</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCapacity">
       <property name="text">
        <string>ظرفیت</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spnCapacity">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDepartDate">
       <property name="text">
        <string>تاریخ رفت</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="HLayoutDepartDate">
       <item>
        <widget class="QPushButton" name="btnDepartDatePicker">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/resources/icons/calendar.png</normaloff>:/icons/resources/icons/calendar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDepartDate"/>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="HLayoutDepartTime">
       <item>
        <widget class="QSpinBox" name="spnDepartMinute">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDepartTimeColon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnDepartHour">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblReturnDate">
       <property name="text">
        <string>تاریخ برگشت</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="HLayoutReturnDate">
       <item>
        <widget class="QPushButton" name="btnReturnDatePicker">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/resources/icons/calendar.png</normaloff>:/icons/resources/icons/calendar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtReturnDate"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblDepartTime">
       <property name="text">
        <string>ساعت رفت</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblReturnTime">
       <property name="text">
        <string>ساعت برگشت</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="HLayoutReturnTime">
       <item>
        <widget class="QSpinBox" name="spnReturnMinute">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDepartTimeColon_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnReturnHour">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbOrigin">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbDestination">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>خطا</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>ایجاد اردو</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/resources/icons/addTour.png</normaloff>:/icons/resources/icons/addTour.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReturn">
       <property name="text">
        <string>بازگشت</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/resources/icons/return.png</normaloff>:/icons/resources/icons/return.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbOrigin</tabstop>
  <tabstop>cmbDestination</tabstop>
  <tabstop>spnCapacity</tabstop>
  <tabstop>btnDepartDatePicker</tabstop>
  <tabstop>txtDepartDate</tabstop>
  <tabstop>spnDepartHour</tabstop>
  <tabstop>spnDepartMinute</tabstop>
  <tabstop>btnReturnDatePicker</tabstop>
  <tabstop>txtReturnDate</tabstop>
  <tabstop>spnReturnHour</tabstop>
  <tabstop>spnReturnMinute</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>btnReturn</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
